.github/workflows/go.yml

```
@@ -0,0 +1,25 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+
+    - name: Build
+      run: go build -v ./...
```

Uhmm, essa etapa pode ser redundante se a gente já está executando o `go test` já que ele também compila o código. Teve também uma coisa que eu senti falta, uma chamada ao `go vet ./...`

O que você acha de a gente fazer uma mudança assim:

```suggestion
    - name: Lint
      run: go vet ./...
```